docs/pages/web/numberfield.js

```diff
@@ -556,6 +556,37 @@ function Example(props) {
           />
         </MainSection.Subsection>
 
+        <MainSection.Subsection
+          title="EnterKeyHint"
+          description={`The \`enterKeyHint\` prop presents to the user a more accurate action label for the enter key on virtual keyboards. These are the values for each use case:
+
+- "enter": typically used for inserting a new line
```

Comment:
"Typically used" is pretty repetitive here — perhaps we can leave it off?

Reply:
DONE